databricks/koalas/frame.py

```diff
@@ -6425,8 +6425,16 @@ def _pd_getitem(self, key):
             return DataFrame(self._internal.copy(sdf=self._sdf.filter(bcol)))
         raise NotImplementedError(key)
 
+    def _to_pandas(self):
```

Comment:
Actually, how about naming it explicitly like ... `_to_internal_pandas` or .. `_to_pandas_without_copy` ... There is kiiiiind off ... similar one I did `spark_internal_df` vs `spark_df` for instance FYI.